package.json

```diff
@@ -9,8 +9,11 @@
     "local-build-addon": "rollup -c rollup.config.addon.js --config-study-list-url=/public/locally-available-studies.json",
     "test-addon": "mocha --require  \"./tests/core-addon/hooks.js\" \"./tests/core-addon/*.js\"",
     "lint-addon": "web-ext lint",
+    "lint-js": "eslint .",
+    "lint": "npm run lint-addon && npm run lint-js",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test": "npm run test-addon"
```

Comment:
I slightly changed #27 , because we now have a confusing explosion of commands :)